cypress/support/pages/signIn.pageObject.js

```diff
@@ -25,6 +25,15 @@ class SignInPageObject extends PageObject {
       .type(password);
   }
 
+  fillLoginCredentials(email, password) {
+    if (email) {
```

Comment:
You have not provided any feedback if the email or password is not provided. It would be better to throw an error or return a message indicating that email or password is required.

Reply:
Done